lib/sdam/topology_description.js

```diff
@@ -284,11 +284,11 @@ function topologyTypeForServerType(serverType) {
       return TopologyType.Sharded;
     case ServerType.RSPrimary:
       return TopologyType.ReplicaSetWithPrimary;
-    case ServerType.RSGhost:
-    case ServerType.Unknown:
-      return TopologyType.Unknown;
-    default:
+    case ServerType.RSOther:
+    case ServerType.RSSecondary:
       return TopologyType.ReplicaSetNoPrimary;
```

Comment:
It didn't make sense to me to have the "default" be `ReplicaSetNoPrimary` so I swapped this, initial spec tests showed this worked.

Reply:
nit: I don't find that using a `switch` here improves readability at all :man_shrugging: Maybe adding some line breaks in there would be useful, I just find it hard to visually track

Reply:
I generally don't prefer switch syntax, but i think @emadum's recommendation makes sense here it does help restrict what this function can do. It makes it more obvious that this function takes a `ServerType` and returns a `TopologyType` and no other logic is really allowed.

Reply:
default `Unknown` works for me, I'm fine to resolve this comment